contracts/utils/cryptography/ECDSA.sol

```diff
@@ -22,6 +22,10 @@ library ECDSA {
      * recover to arbitrary addresses for non-hashed data. A safe way to ensure
      * this is by receiving a hash of the original message (which may otherwise
      * be too long), and then calling {toEthSignedMessageHash} on it.
+     *
+     * Documentation for signature generation:
```

Comment:
It would be nicer to show the code inline here. "A client would generate signatures by using the signing methods in Web3.js or Ethers.js, as shown below:" ...